Revisit prob config drop down deletion

On dev where we have the changes made by this pr https://github.com/StatCan/aaw-kubeflow-profiles-controller/pull/135 we are unable to create a protected b notebook with a volume attached. Note that this image isn't on prod and we can create just fine. 


See [dev](https://github.com/StatCan/aaw-argocd-manifests/blob/aaw-dev-cc-00/daaas-system/profile-controllers/application.jsonnet) vs  [prod](https://github.com/StatCan/aaw-argocd-manifests/blob/aaw-prod-cc-00/daaas-system/profiles-controller/application.jsonnet) and changing the image sha to use what's on prod allowed us to schedule again.

When version 46,  which only removed the poddefault is in use;
`Reissued from statefulset/version46-no-default: create Pod version46-no-default-0 in StatefulSet version46-no-default failed error: admission webhook "validation.gatekeeper.sh" denied the request: [disk-data-classification] volume <version46-no-default-volume> data classification <protected-b> conflicts with pod <version46-no-default-0> data classification <unclassified>`

--- a/kustomize/apps/jupyter-web-app/base/deployment.yaml
+++ b/kustomize/apps/jupyter-web-app/base/deployment.yaml
@@ -9,7 +9,7 @@ spec:
       containers:
       - name: jupyter-web-app
         imagePullPolicy: IfNotPresent
-        image: k8scc01covidacr.azurecr.io/jupyter-apis:9160bf5ad24f0aecccf0b3e5ff670b57f62ab0dc
+        image: k8scc01covidacr.azurecr.io/jupyter-apis:6c4b579c25ac1904fb6087e0f1cd326a8e04376e
         env:
         - name: UI
           value: default
@@ -57,7 +57,7 @@ spec:
       initContainers:
       - name: copy-jwa-static-files
         # Change for the image I just built
-        image: k8scc01covidacr.azurecr.io/jupyter-apis:9160bf5ad24f0aecccf0b3e5ff670b57f62ab0dc
+        image: k8scc01covidacr.azurecr.io/jupyter-apis:6c4b579c25ac1904fb6087e0f1cd326a8e04376e
         # image: k3d-s3proxy-registry:5050/s3proxy-dev:latest
         command: [sh, -c]
         args: ["cp -r /src/dist/frontend/* /etc/nginx/html/"]

